ci-operator/step-registry/ovn/conf/vsphere-dualstack/ovn-conf-vsphere-dualstack-ref.yaml

```diff
@@ -9,5 +9,8 @@ ref:
     requests:
       cpu: 10m
       memory: 100Mi
+  env:
+    - name: IP_FAMILIES
+      default: "DualStackIPv6Primary"
```

Comment:
I am not sure about this part... It modifies the `ovn-conf-vsphere-dualstack` step which may be used by a lot of jobs and it sets the default family to `DualStackIPv6Primary` which doesn't have to be true because there are dual-stack jobs with IPv4-primary. Wouldn't it be better to not have a default at the step level and just set it for the specific job?

Reply:
It's a question of coverage. Every other job tests IPv4.  This is our **_only_** IPv6 primary test?  We need at least 75% of vSphere testing to be with IPv6.

OPNET-198 is done.  IPv6 primary is supported.

Reply:
I haven't figured out how to handle the 4.13 jobs yet.